### PR TITLE
List view w items

### PR DESCRIPTION
**Pull request by regnisalram**
dummy data added to adapter (with title and description, nothing else yet)
but fragments shows listItem titles

### CODE REVIEW

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java [hunk 1]
```diff
@@ -63,6 +59,10 @@ public void login(View view){
             @Override
             public void onResponse(Call<UserResponse> call, Response<UserResponse> response) {
                 UserResponse user = response.body();
+                editor.putString("userName", username);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java:72]: @regnisalram ask for the sharedpreferences when you need it, so here not above

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/server/MockServer.java [hunk 2]
```diff
@@ -14,6 +17,11 @@
 
 public class MockServer implements ReaderApiInterface {
     @Override
+    public Call<ArrayList<NewsItem>> getNewsItems() {
+        return null;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/server/MockServer.java]: @regnisalram create a mock version for this API call as well, and return a dummy list just as a dummy user is returned below

File: app/src/main/res/layout/newsitem.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        android:text="TextView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/title" />
+
+    <!--<TextView-->
+        <!--android:text="TextView"-->
+        <!--android:layout_width="match_parent"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:id="@+id/description" />-->
```
Comment (adamgyulavari) [inline app/src/main/res/layout/newsitem.xml]: @regnisalram dont push commented code

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java (hunk 1 above)
Comment (regnisalram) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java:72]: can't move it here, in ```PreferenceManager.getDefaultSharedPreferences(this);``` this gets a red line and i don't know why.

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/server/MockServer.java (hunk 2 above)
Comment (regnisalram) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/server/MockServer.java]: DOOOOONNNNEEEEEEEEE!!!!!!!!!!!!!!!!!!!!!!!!!

File: app/src/main/res/layout/newsitem.xml (hunk 3 above)
Comment (regnisalram) [reply app/src/main/res/layout/newsitem.xml]: ok.

**Pull request merged**